tests/dataframe_test.py

```diff
@@ -387,6 +387,11 @@ def test_gb_series(self):
         self.assertListEqual(c.index.to_list(), ["Bob", "Alice", "Carol"])
         self.assertListEqual(c.values.to_list(), [2, 3, 1])
 
+        # testing counts with IPv4 column
+        s = ak.DataFrame({'a': ak.IPv4(ak.array([1, 2, 3, 4]))}).groupby('a').count()
```

Comment:
Small style suggestion, change to ak.arange(1, 4)

Reply:
@jaketrookman - I agree with the use of `ak.arange` here. I just copied over the user provided reproducer originally. However, when using `ak.arange`, the stop parameter is exclusive, so we would need `ak.arange(1,5)`.